.github/workflows/firebase-hosting-pull-request.yml

```
@@ -29,5 +31,6 @@ jobs:
           REACT_APP_FIREBASE_APP_ID: ${{ secrets.REACT_APP_FIREBASE_APP_ID }}
       - uses: FirebaseExtended/action-hosting-deploy@v0
         with:
+          repoToken: "\${{ secrets.GITHUB_TOKEN }}"
```

actionlint (1.7.12)

[error] could not parse as YAML: found unknown escape character

(syntax-check)